### PR TITLE
Clean up / Lint

### PR DESCRIPTION
**Pull request by ShoGinn**
Again.. I was using 2 containers for this.. much cleaner and resource friendly!

### CODE REVIEW

**Comment by mikenye** [comment]
Thanks so much for this! What linter are you using?

**Comment by ShoGinn** [comment]
Vscode with plugins. I use it for python and it works awesome compared to sublime etc. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: README.md [hunk 1]
```diff
@@ -48,15 +56,15 @@ The latitude and longitude of your antenna must be passed via the `LAT` and `LON
 
 The altitude of your antenna must be passed via the `ANT` environment variable respectively. Defaults to metres, but units may specified with a 'ft' or 'm' suffix.
 
-Lastly, you should specify a site name via the `SITENAME` environment variable. This field supports letters, numbers, `-` & `_` only. Any other characters will be stripped upon container initialisation.
+Lastly, you should specify a site name via the `SITENAME` environment variable. This field supports letters, numbers, `-` & `_` only. Any other characters will be stripped upon container initialization.
```
Comment (mikenye) [inline README.md]: I'm in Australia, so for us it is "initialisation", however I'll let it slide. :)

**Comment by mikenye** [comment]
OK this is great stuff. I'm going to merge this. Thanks for your hard work.

**Comment by mikenye** [comment]
> Vscode with plugins. I use it for python and it works awesome compared to sublime etc.

Thanks for the info. I've just started using Atom (I'm on a mac) so will see if I can find some good linters to tidy up my code. :)

**Comment by ShoGinn** [comment]
Also using a Mac. Ever since switching to vscode I’ve been very happy. Tons of plugins and formatters. I originally started using it because of how awesome its python debugging is.  But the text editor and overall project management is just easy in it. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**